bcgov/moh-prime
# PRIME-2054 Do not persist SD documents when answer changes from Yes to No

[PRIME-2054](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-2054)

### How to test:

1. Create an enrolment and go through the steps until you reach the Self Declaration page
2. Answer Yes to any of the questions, add description and upload a document.
3. Go to overview (You should see the document under the Self Declaration section under the question that has a **Yes** answer)
4. Click the edit button on the Self Declaration section in overview (You should be taken back to Self Declaration page and see the uploaded document there)
5. Change the answer from **Yes** to **No**
6. Go to overview (The document should be displayed anymore)
7. Go back to Self Declaration (The document should be displayed)
8. Change the answers to Yes to all questions (No document should be displayed)
9. Login as admin, and view the enrolment (You should see the document under the Self **Declaration** section in the **Enrolment** card)
